nativeauthenticator/nativeauthenticator.py

```diff
@@ -69,6 +69,11 @@ class NativeAuthenticator(Authenticator):
         default_value=False,
         help="Deletes FirstUse Authenticator database after the import"
     )
+    add_two_factor_authentication = Bool(
```

Comment:
Can we split this into two parts:

1. *Allow* users to have 2FA
2. *Require* users to have 2FA

In the former, some users might choose to protect their own accounts with 2FA, but isn't enforced. (2) is more complicated - what happens to people who already had created accounts without 2FA? How can they log in?

So I think one PR should be to allow_two_factor_authentication (or just allow_2fa), and it makes it optional for users to set up and use 2FA. There should probably also be a button in the admin screen that shows admins if this user has 2fa enabled, and optionally allows them to reset it.

After that, we can figure out how to make it required.

Reply:
Indeed this makes sense. I'll refactor this PR to make the 2fa an optional character within user and then make another one to make it required and how to deal with people that signed in without 2fa, ok?